### PR TITLE
ReverseString

### PR DESCRIPTION
**Pull request by arpi-r**
Issue #9

This Pull Request resolves Issue #<IssueNumber>

Description:
In this Pull Request we have done the following additions/edits for the code:

Note: Delete all the text including this!
* Make sure you replace `#<IssueNumber>` with the particular Issue that you are referencing with!
* Make sure you describe the Issue properly and ask as many people as possible to review your Pull Request so that it can be merged as soon as possible!

### CODE REVIEW

File: ReverseString [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#include<stdio. h>
```
Comment (MJ10) [inline ReverseString]: Please move this file to the `reverse` directory and rename it to your Github username with the `.c` extension

File: ReverseString.c [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#include<stdio. h>
```
Comment (MJ10) [inline ReverseString.c]: Move the file to the `ReverseString` directory and rename it to your github username

**Comment by MJ10** [comment]
@arpi-r any updates?

**Comment by salman-bhai** [comment]
Are there any updates or should we close the PR?

**Pull request closed without merging**